scripts/zones/Vunkerl_Inlet_[S]/mobs/Pallass_Tiger.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Vunkerl Inlet [S]
+--  Mob: Pallas's Tiger
+-----------------------------------
+local ID = zones[xi.zone.VUNKERL_INLET_S]
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.LULLABYRES, 100)
```

Comment:
that's not how we make things immune, which looks like what you want since you set that clear at 100.

immunity is a bit field in mob pools (add up numbers of statuses from the immunity enum that you want and stick resulting number in the pool field)